Xamarin.Essentials/TextToSpeech/TextToSpeech.android.cs

```diff
@@ -0,0 +1,290 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Android.OS;
+using Android.Speech.Tts;
+using Xamarin.Essentials;
+
+namespace Xamarin.Essentials
+{
+    public static partial class TextToSpeech
+    {
+        const int maxSpeechInputLengthDefault = 4000;
+
+        static global::Android.Speech.Tts.TextToSpeech textToSpeech;
+
+        static TextToSpeech()
+        {
+            Task.Run(() => Initialize());
+
+            return;
+        }
```

Comment:
I am scared about this, there may be potential race conditions. This is what I did for flashlight that needed an init: https://github.com/xamarin/Essentials/blob/master/Xamarin.Essentials/Flashlight/Flashlight.uwp.cs

Reply:
Same, if you look what I did is have an Init() method that handles all of it: https://github.com/jamesmontemagno/TextToSpeechPlugin/blob/master/src/TextToSpeech.Plugin.Android/TextToSpeech.cs